### PR TITLE
Validators: parameterized string type

### PR DESCRIPTION
**Pull request by rjbs**

### CODE REVIEW

File: lib/Ix/DBIC/Result.pm [hunk 1]
```diff
@@ -183,8 +183,8 @@ sub ix_add_properties ($class, @pairs) {
 
 my %DEFAULT_VALIDATOR = (
   integer => Ix::Validators::integer(),
-  string  => Ix::Validators::simplestr(),
-  istring => Ix::Validators::simplestr(),
+  string  => Ix::Validators::string({ oneline => 1 }),
+  istring => Ix::Validators::string({ oneline => 1 }),
```
Comment (wolfsage) [inline lib/Ix/DBIC/Result.pm:187]: simplestr made sure our strings weren't whitespace only. We don't appear to do that anymore. Is that intentional?

Comment (wolfsage) [reply lib/Ix/DBIC/Result.pm]: Looks good minus one q

**Comment by wolfsage** [comment]
Looks good minus 1 ?

**Comment by mmcclimon** [comment]
I'd forgotten this was in my other inbox...I second what alh said.

File: lib/Ix/DBIC/Result.pm (hunk 1 above)
Comment (rjbs) [reply lib/Ix/DBIC/Result.pm:187]: `string()` has the `printable` option on by default.  `printable` requires the string match `/\pL\pN\pP\pS/`, which will exclude whitespace-only characters.  You can see the test for config `{ nonempty => 1 }` working like this.

**Comment by rjbs** [comment]
Well merge.

**Pull request closed without merging**